bigtable/tests/mutations_integration_test.cc

```diff
@@ -0,0 +1,555 @@
+                           time_end);
+
+  auto actual_cells = ReadRows(*table, bigtable::Filter::PassAllFilter());
+
+  DeleteTable(table_name);
+
+  CheckEqualUnordered(expected_cells, actual_cells);
+}
+
+/**
+ * Verify that records deleted for a specific family will delete correct
+ * records for that family only
+ */
+TEST_F(MutationIntegrationTest, DeleteFromFamilyTest) {
+  std::string const table_name = "table-delete-for-family";
+
+  auto table = CreateTable(table_name, table_config);
+
+  // Create a vector of cell which will be inserted into bigtable
+  std::string const row_key = "DelColKey";
+  std::vector<bigtable::Cell> created_cells{
+      {row_key, column_family1, "column_id1", 0, "v-c-0-0", {}},
+      {row_key, column_family1, "column_id1", 1000, "v-c-0-1", {}},
+      {row_key, column_family2, "column_id3", 2000, "v-c-0-2", {}},
+      {row_key, column_family2, "column_id2", 2000, "v-c0-0-0", {}},
+      {row_key, column_family1, "column_id3", 3000, "v-c1-0-2", {}},
+  };
+
+  std::vector<bigtable::Cell> expected_cells{
+      {row_key, column_family2, "column_id3", 2000, "v-c-0-2", {}},
+      {row_key, column_family2, "column_id2", 2000, "v-c0-0-0", {}},
+  };
+
+  // Create records
+  CreateCells(*table, created_cells);
+
+  // Delete all the records for family
+  DeleteFromFamily(*table, row_key, column_family1);
+
+  auto actual_cells = ReadRows(*table, bigtable::Filter::PassAllFilter());
+
+  DeleteTable(table_name);
+
+  CheckEqualUnordered(expected_cells, actual_cells);
+}
+
+/**
+ * Verify that records deleted for a specific row will delete correct
+ * records for that row only
+ */
+TEST_F(MutationIntegrationTest, DeleteFromRowTest) {
+  std::string const table_name = "table-delete-for-row";
+
+  auto table = CreateTable(table_name, table_config);
+
+  // Create a vector of cell which will be inserted into bigtable
+  std::string const row_key1 = "DeleteRowKey1";
+  std::string const row_key2 = "DeleteRowKey2";
+  std::vector<bigtable::Cell> created_cells{
+      {row_key1, column_family1, "column_id1", 0, "v-c-0-0", {}},
+      {row_key1, column_family1, "column_id1", 1000, "v-c-0-1", {}},
+      {row_key1, column_family2, "column_id3", 2000, "v-c-0-2", {}},
+      {row_key2, column_family2, "column_id2", 2000, "v-c0-0-0", {}},
+      {row_key2, column_family3, "column_id3", 3000, "v-c1-0-2", {}},
+  };
+
+  std::vector<bigtable::Cell> expected_cells{
+      {row_key2, column_family2, "column_id2", 2000, "v-c0-0-0", {}},
+      {row_key2, column_family3, "column_id3", 3000, "v-c1-0-2", {}},
+  };
+
+  // Create records
+  CreateCells(*table, created_cells);
+
+  // Delete all the records for a row
+  DeleteFromRow(*table, row_key1);
+
+  auto actual_cells = ReadRows(*table, bigtable::Filter::PassAllFilter());
+
+  DeleteTable(table_name);
+
+  CheckEqualUnordered(expected_cells, actual_cells);
+}
+// Test Cases Finished
+
+int main(int argc, char* argv[]) try {
```

Comment:
Please note that we stopped wrapping integration tests with `try/catch` blocks.  That is why your build broke (one of the reasons at least).